src/content/CloudBoilerplateNet/Services/CachedDeliveryClient.cs

```diff
@@ -297,6 +296,31 @@ protected virtual void Dispose(bool disposing)
             _disposed = true;
         }
 
+        public Task<JObject> GetTaxonomyJsonAsync(string codename)
```

Comment:
GetTaxonom* methods should, similarly to other Get* methods, create a cache key and call the original client through `GetOrCreateAsync`

Reply:
@petrsvihlik I have included above changes . Please review .